### PR TITLE
Rusty Fix

### PR DESCRIPTION
**Pull request by emilyons**
fixes issue with product create missing

### CODE REVIEW

**Comment by emilyons** [comment]
I see that you changed this. Why?

**Comment by vixdug** [comment]
🚢

**Review by vixdug — APPROVED** [review]
Good find.

File: hackdays/db/seeds.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@
 )
 
 
-
+Product.create(
```
Comment (gnarlynate) [inline hackdays/db/seeds.rb:59]: Looks good to me

**Review by gnarlynate — APPROVED** [review]

**Review by colinthecannon — APPROVED** [review]
😄

**Pull request merged**